manifests/kustomize/base/params.env

```diff
@@ -6,3 +6,4 @@ mlmdDb=metadb
 cacheDb=cachedb
 pipelineDb=mlpipeline
 bucketName=mlpipeline
+containerRuntimeExecutor=docker
```

Comment:
Other params not documented yet, but let's start adding some doc for this.

You can use # for comment.

We should have a link to the container runtime executor documentation at least

Reply:
@Bobgy  I wonder do we want to have a section in the official documentation for this? Beside https://github.com/kubeflow/website/pull/2056 I was wondering if make sense extend the pipeline standalone docs and mention this for all other configurations.
I can make the doc extension in case.

Reply:
Sure, that's a good idea, it can probably be a topic under customizing kfp.
Other params here and some undocumented ones can go into the same document

Reply:
Will prepare the doc and PR to this :)

Reply:
added a few raws btw